manifests/ca.pp

```diff
@@ -40,8 +40,8 @@
 define trusted_ca::ca (
   Optional[String] $source = undef,
   Optional[Pattern['^[A-Za-z0-9+/\n=-]+$']] $content = undef,
-  Stdlib::Absolutepath $install_path = $trusted_ca::install_path,
-  String $certfile_suffix = $trusted_ca::certfile_suffix,
+  Stdlib::Absolutepath $install_path                 = $trusted_ca::install_path,
```

Comment:
Please don't align class parameters. It's unrelated to the actual fix and just introduces noise.